src/Microsoft.Data.Sqlite.Core/SqliteCommand.cs

```diff
@@ -180,14 +174,23 @@ protected override DbParameterCollection DbParameterCollection
         public override UpdateRowSource UpdatedRowSource { get; set; }
 
         /// <summary>
+        /// Gets or sets the data reader currently being used by the command, or null if none.
+        /// </summary>
+        /// <value>The data reader currently being used by the command.</value>
+        protected internal virtual SqliteDataReader DataReader { get; set; }
+
+        /// <summary>
         /// Releases any resources used by the connection and closes it.
         /// </summary>
         /// <param name="disposing">
         /// true to release managed and unmanaged resources; false to release only unmanaged resources.
         /// </param>
         protected override void Dispose(bool disposing)
         {
-            DisposePreparedStatements();
+            if (disposing)
```

Comment:
Before switching to SQLitePCL.raw, this case was handled by using `SafeHandle` objects for all the the native handles. All our code currently assumes this behavior. I'll follow up with a new issue to investigate how we need to react now that we're using SQLitePCL.raw